File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int number, i, j, k;
    do
    {
        number = get_int("Height:\n");
    }
    while (number < 1);
    for (i = 1; i <= number; i++)
    {
        for (j = 1; j <= number; j++)
        {
            k = number - 1;
            if (j <= k)
            {
                printf(" ");
            }
            else
                printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-less/mario.py
from cs50 import get_int
t = get_int("Height: ")
while t > 8 or t < 0:
    t = get_int("Height: ")

for p in range(1, t + 1, 1):
    s = t - p
    for _ in range(s):
        print(" ", end = "");
    for _ in range(s, t , 1):
        print("#", end = "")
    print()


